### ISSUE TITLE
🐞 Bug report: Element IDs are not unique - Accessibility issue (urgent)

### ISSUE THREAD

**Issue by yanemr** [elementor/elementor#19137]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Hello, We are running out of time to meet government deadlines for Accessibility fixes. We were able to fix some of the issues inside the Elementor plugin using our functions.php and a .js file, but we can not find a way to fix the issue (Element IDs are not unique). We don't like the idea of doing something inside the plugin code, because we know we will lose our changes when a release comes. Also, we are not sure why Elementor is using this id="elementor-device-mode".  We were planning to remove it, but we are not 100% sure about the function it has. Could someone help us, please? We were alerted about the issue because of the Siteimprove platform. We can provide screenshots from the platform with more information needed. We will be waiting to hear from you. Thank you in advance.

### Steps to reproduce

1- Going to childrenbelieve.ca
2- Using chrome, use the "developer tools" option to inspect the homepage
3- In the "Elements" tab, going up from the bottom of the the html code, you will see ```<span id="elementor-device-mode" class="elementor-screen-only"></span>```, and the code below:

```
<span id="elementor-device-mode" class="elementor-screen-only"></span>
<svg class="e-font-icon-svg-symbols" style="display: none;"></svg>
<svg class="e-font-icon-svg-symbols" style="display: none;"></svg>
<span id="elementor-device-mode" class="elementor-screen-only"></span>
</body>
</html>
```

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by m1ga** [comment]
duplicate of https://github.com/elementor/elementor/issues/17487

**Comment by yanemr** [comment]
Hello, in ticket #17487 we have three issues. We found later that the "Role not inside the required context" issue is coming from another plugin, not from the Elementor plugin and it is fixed. And, as we mentioned above, we were able to fix the "Hidden element has focusable content" that is from the Elementor Pro plugin, using our theme. I hope this helps and we really hope that the Elementor developer team fixes the "Element IDs are not unique" issue soon. Thank you

**Comment by yanemr** [comment]
So, only confirming, do you think we can remove it or assign a different id to all the elements? That will be great news. And, yes, we will try as we did with the other issues, but as we are using an icons library, that looks like is connected to that, we think that the Elementor Dev team should be the one fixing the issue. We will try then, to pass our tests and we hope the dev team will fix the plugin in the future. Thanks!

**Comment by m1ga** [comment]
I just say you should try it. Looking at my page I only see one `elementor-device-mode` element and that contains the screensize I'm using at the moment. In your code is a `RocketElementorAnimation` that adds another one of those. So maybe its wp rockets fault and that adds a second element? I'm just guessing here but that was what I saw when quickly searching for `elementor-device-mode` in your code

**Comment by yanemr** [comment]
Yes, that is what we saw too and that's why we want to know if the Elementor is using this id or not to remove it. We will try to fix it on our end. Thank you so much!

**Comment by rami-elementor** [comment]
I found `elementor-device-mode` in Elementor code only once:
https://github.com/elementor/elementor/blob/master/assets/dev/js/frontend/frontend.js#L67


I also checked your site https://childrenbelieve.ca/

As @m1ga mentioned, it's not Elementor's code. It comes from WP Rocket

```html
<!-- This website is like a Rocket, isn't it? Performance optimized by WP Rocket. Learn more: https://wp-rocket.me - Debug: cached@1657661095 -->
<span id="elementor-device-mode" class="elementor-screen-only"></span>
<svg style="display: none;" class="e-font-icon-svg-symbols"></svg>
<svg style="display: none;" class="e-font-icon-svg-symbols"></svg>
<span id="elementor-device-mode" class="elementor-screen-only"></span>
```

Closing this issue, as it's not an Elementor issue. Please contact WP Rocket.

**Comment by yanemr** [comment]
Thank you so much @m1ga for taking the time to do the deep check out of this issue, and thank you so much @rami-elementor for confirming that it looks like this issue is not coming from the Elementor plugin. Yes, we can close this ticket and ticket #17487 as well. Again, thank you!
